cesium/build_model.py

```diff
@@ -148,6 +148,10 @@ def build_model_from_featureset(featureset, model=None, model_type=None,
         if inf_feats.any():
             message += " Inf in {}.".format(", ".join(feature_df.columns[inf_feats]))
         raise ValueError(message)
+    try:
+        model = model.best_estimator_
```

Comment:
What's the advantage of returning this over the `GridSearchCV` object? I'm a little unclear on how that class works but I thought it let you `fit` and `predict` which seems like all you need...?